index.html

```diff
@@ -315,6 +315,52 @@ <h3>
             </div>
         </div>
     </section>
+    <!-- form section -->
+    <section class="form-section">
+        <article class="form-article">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </article>
+        <form novalidate action="https://formspree.io/f/xqkjrlry" method="post">
+
+            <input type="text" id="full-name" class="form-controll" maxlength="30" name="full_name"
+                placeholder="First name" required>
+            <input type="email" id="email" class="form-controll" name="user-email" placeholder="Email address"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required />
+            <textarea name="message" maxlength="500" placeholder="Enter message here" required></textarea>
+            <button class="form_button" type="submit">Get in touch</button>
+        </form>
+    </section>
+    <hr>
+    <footer>
+        <div class="icons">
+            <ul>
+                <li>
+                    <a href="#">
+                        <img src="./images/Icon -GitHub.png" alt="github" /></a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Icon - Linkedin.png" alt="Linkedin" />
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Icon - Angelist.png" alt="Angelist" />
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Icon - Twitter.png" alt="Twitter" />
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <img src="./images/Icon - Medium.png" alt="Medium" />
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </footer>
```

Comment:
Well done! 

1. As I mentioned above it's highly recommended to follow the exact Figma design, so please make sure to use the same color for your icons.
2. The bottom line is missing in your design, kindly make sure to add it as well.

Yours:
![image](https://user-images.githubusercontent.com/34779928/206286277-951bd0c3-8efd-4d30-85d0-46f1f7d93639.png)

Original:
![image](https://user-images.githubusercontent.com/34779928/206286320-a69fc802-4b0a-4cf0-8019-c900d6d536fc.png)